Misc/valgrind-python.supp

```diff
@@ -7,8 +7,8 @@
 #	valgrind --tool=memcheck --suppressions=Misc/valgrind-python.supp \
 #		./python -E ./Lib/test/regrtest.py -u gui,network
 #
-# You must edit Objects/obmalloc.c and uncomment Py_USING_MEMORY_DEBUGGER
-# to use the preferred suppressions with address_in_range.
+# You must edit Objects/obmalloc.c to use the preferred
+# suppressions with address_in_range.
```

Comment:
You should remove the whole sentence.

I don't understand this sentence. The command already uses `--suppressions=Misc/valgrind-python.supp`.

The sentence was there only to uncomment Py_USING_MEMORY_DEBUGGER.

Reply:
Honestly, I was confused by it as well... I'll remove it